mpsl/README.rst

```diff
@@ -0,0 +1,21 @@
+.. _mpsl:
+
+MPSL
+====
+
+Multi-Protocol Service Layer
+
```

Comment:
Add some introduction text here describing the main purpose and functionality of the libarary?

Reply:
Can add something like this for now: "Multi-Protocol Service Layer (MPSL) is a library of common services for single and multi-protocol wireless implementations."
But we should add more stuff here later.

Reply:
Done